### PR TITLE
feat: add policy type to metadata

### PR DESCRIPTION
**Pull request by fabriziosestito**
## Description
Adds `policyType` (kubernetes/raw, defaults to kubernetes) to the Policy metadata.

## Test

yes

### CODE REVIEW

**Review by viccuad — APPROVED** [review]
LGTM. I would prefer to explicitly add `policyType: kubernetes` to the `metadata_backwards_compatibility` line 385.

File: src/policy_artifacthub.rs [hunk 1]
```diff
@@ -509,6 +509,7 @@ mod tests {
             background_audit: true,
             context_aware_resources: HashSet::new(),
             execution_mode: Default::default(),
+            policy_type: Default::default(),
```
Comment (flavio) [inline src/policy_artifacthub.rs:512]: I think we should be explicit and make it `PolcyType::Kubernetes`

**Review by flavio — COMMENTED** [review]
LGTM, but I agree with @viccuad's comment

File: src/policy_artifacthub.rs (hunk 1 above)
Comment (fabriziosestito) [reply src/policy_artifacthub.rs:512]: done in https://github.com/kubewarden/policy-evaluator/pull/368/commits/8086300c601b4b29b1d5cac15213381b17bb88eb

**Pull request merged**